ultron-compose/src/androidMain/kotlin/com/atiurin/ultron/core/compose/view/ComposeInViewExt.kt

```diff
@@ -0,0 +1,24 @@
+package com.atiurin.ultron.core.compose.view
+
+import android.view.View
+import androidx.compose.ui.platform.AbstractComposeView
+import androidx.compose.ui.test.InternalTestApi
+import androidx.compose.ui.test.SemanticsMatcher
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.createTestContext
+
+/**
+ * Select Compose Node by [matcher] in a particular View
+ */
+@OptIn(InternalTestApi::class)
+fun View.semanticsNodeInteraction(
```

Comment:
lets rename this method to `getSemanticsNodeInteraction` and return ultron wrapper
```kotlin
@OptIn(InternalTestApi::class)
fun View.getSemanticsNodeInteraction(
    matcher: SemanticsMatcher,
    useUnmergedTree: Boolean = true
): UltronComposeSemanticsNodeInteraction {
    val composeView = requireNotNull(this as? AbstractComposeView) { "View is not subtype of AbstractComposeView. It is ${this::class.java}" }
    return UltronComposeSemanticsNodeInteraction(SemanticsNodeInteraction(
        createTestContext(ViewAndroidTestOwner(composeView)),
        useUnmergedTree,
        matcher
    ))
}
```